js/admin.js

```diff
@@ -7,6 +7,23 @@
 			mode: 'shell'
 		} );
 
+	function checkForAdsFile(){
+		var currentTime = (new Date()).getTime();
```

Comment:
This can be simplified a bit
```suggestion
		var currentTime = Date.now();
```